TabloidCLI/Repositories/NoteRepository.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Data.SqlClient;
+using TabloidCLI.Models;
+using TabloidCLI.Repositories;
+using TabloidCLI.UserInterfaceManagers;
+using System.Data;
+
+namespace TabloidCLI.Repositories
+{
+    public class NoteRepository : DatabaseConnector, IRepository<Note>
+    {
+        public NoteRepository(string connectionString) : base(connectionString) { }
+
+        public List<Note> GetAll()
+        {
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"SELECT Id, Title, Content, CreateDateTime, PostId FROM Note";
+
+                    List<Note> notes = new List<Note>();
+
+                    SqlDataReader reader = cmd.ExecuteReader();
+                    while (reader.Read())
+                    {
+                        Note note = new Note()
+                        {
+                            Id = reader.GetInt32(reader.GetOrdinal("Id")),
+                            Title = reader.GetString(reader.GetOrdinal("Title")),
+                            Content = reader.GetString(reader.GetOrdinal("Content")),
+                            CreateDateTime = reader.GetDateTime(reader.GetOrdinal("CreateDateTime")),
+                            //note.PostId = reader.GetInt32(reader.GetOrdinal("note.PostId"))
+
+                            //Post = new Post()
+                            //{
+                            //    Id = reader.GetInt32(reader.GetOrdinal("postId")),
+                            //    Title = reader.GetString(reader.GetOrdinal("postTitle")),
+                            //    Url = reader.GetString(reader.GetOrdinal("postURL"))
+                            //}
+                        };
+                        notes.Add(note);
+                    }
+                    reader.Close();
+                    return notes;
+                }
+            }
+        }
+
+        public Note Get(int id)
+        {
+            throw new NotImplementedException();
+        }
+
+        //public List<Note> GetByPost(int postId)
```

Comment:
is this code used for something later? if not please remove any commented and unused methods

Reply:
Code block commented out from lines 57-101 is not needed. I've removed them.